.github/workflows/deploy-website.yml

```diff
@@ -0,0 +1,92 @@
+name: Deploy website to Pages
+
+on:
+  push: # Runs on pushes targeting the default branch
+    branches: [ $default-branch ]
```

Comment:
@naskio the `$default-branch` is not supported currently in Github, you have to list the name of the default branch explicitly.